src/transformers/models/bert/modeling_flax_bert.py

```diff
@@ -1677,7 +1677,7 @@ def __call__(
 class FlaxBertForCausalLM(FlaxBertPreTrainedModel):
     module_class = FlaxBertForCausalLMModule
 
-    def prepare_inputs_for_generation(self, input_ids, max_length, attention_mask: Optional[jnp.DeviceArray] = None):
+    def prepare_inputs_for_generation(self, input_ids, max_length, attention_mask = None):
```

Comment:
Would be great to keep the type hint here but update it as required
```suggestion
    def prepare_inputs_for_generation(self, input_ids, max_length, attention_mask: Optional[jax.Array] = None):
```

Reply:
Thanks Sanchit, I've replaced all instances of `jax.DeviceArray` with `jax.Array`.